### PR TITLE
Update the jupyterhub version for secruity

### PR DESCRIPTION
**Pull request by harshad16**
Update the jupyterhub version for secruity
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

File: overlays/python38/Pipfile.lock [hunk 1]
```diff
@@ -670,7 +670,7 @@
                 "sha256:aaf6ab2fcbc68ab6c4c6ba924b57991a388ccc1c8fae782f2433340a40743dfe",
                 "sha256:cf7bae999005dce012738d09610e69a47acd765797c0365c803f96c21ea2f32b"
             ],
-            "markers": "python_version >= '3.4' and python_version < '4'",
+            "markers": "python_version >= '3.4' and python_full_version < '4.0.0'",
```
Comment (Gregory-Pereira) [inline overlays/python38/Pipfile.lock:673]: Why does this use full version, but we change the (other lockfile)[https://github.com/thoth-station/s2i-minimal-notebook/pull/532/files#diff-b93e76970a00833f87a2cbc331328c72129a6b977ef77567eafbe377338613b3R649] to move away from full versson?

Comment (harshad16) [reply overlays/python38/Pipfile.lock:673]: i m not sure on why , if it feel critical , lets file an issue and address it there.

Comment (Gregory-Pereira) [reply overlays/python38/Pipfile.lock:673]: Doesn't feel critical, we can go ahead and move forward. Has testing bee done with these changes?

Comment (harshad16) [reply overlays/python38/Pipfile.lock:673]: yes i tested it on my system

**Comment by harshad16** [comment]
> Could you explain what prompted us to sometimes use `python_full_version` and sometimes just `python_version`? Does using `full_version` not include any micro versions that meet its requirement?

honestly i m not sure
python_full_version is used if the version is x.y.z format and python_version is used if the version is in x.y format
however i m not sure, how version are sometime picked with different formats, its pipenv component locking it
maybe we would have to find the reason for choosing different format of listing.

**Pull request merged**